src/freq/turbomole.f90

```diff
@@ -0,0 +1,101 @@
+! This file is part of xtb.
+!
+! Copyright (C) 2021 Christoph Plett
+!
+! xtb is free software: you can redistribute it and/or modify it under
+! the terms of the GNU Lesser General Public License as published by
+! the Free Software Foundation, either version 3 of the License, or
+! (at your option) any later version.
+!
+! xtb is distributed in the hope that it will be useful,
+! but WITHOUT ANY WARRANTY; without even the implied warranty of
+! MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+! GNU Lesser General Public License for more details.
+!
+! You should have received a copy of the GNU Lesser General Public License
+! along with xtb.  If not, see <https://www.gnu.org/licenses/>.
+
+!> Implementation of aoforce call for hessian
+
+module xtb_freq_turbomole
+   use xtb_mctc_accuracy, only : wp
+   use xtb_type_environment, only : TEnvironment
+   use xtb_type_molecule, only : TMolecule
+
+   implicit none
+
+   character(len=*), parameter :: source = "aoforce_hessian"
+
+   private
+
+   public :: aoforce_hessian 
+
+contains
+
+subroutine aoforce_hessian(env, mol, h, dipd)
+   use xtb_freq_io, only : writeHessianOut
+   use xtb_io_reader, only : readHessian
+   use xtb_type_reader, only : TReader
+   use xtb_mctc_filetypes, only : fileType
+   use xtb_setparam
+
+   implicit none
+   type(TEnvironment), intent(inout) :: env
+   type(TMolecule), intent(in) :: mol
+   type(TReader) :: reader
+   real(wp), intent(inout) :: h(:,:)
+   real(wp), intent(inout) :: dipd(:,:)
+   integer :: idipd, error
+
+   call wrtm(mol%n,mol%at,mol%xyz) !Overwrite coord with RAM-xyz file
+   call execute_command_line('exec aoforce > job.last2>> /dev/null')
+   call reader%open('hessian')
+   call readHessian(env, mol, h, reader, fileType%tmol)
+   call reader%close
+   call writeHessianOut('hessian',h)
```

Comment:
Is there a reason to write the hessian here directly after reading it?

Reply:
It will overwrite the hessian file in turbomole format with a hessian file in the xtb format. If this is not required, I will change it.

Reply:
This might turn out produce interesting results in case you run a namespaced calculation (`--namespace <prefix>`).

Reply:
I've got your point. It will be removed. Thank you.